app.ts

```diff
@@ -0,0 +1,131 @@
+enum Choice {
+    Rock = 'Rock',
+    Paper = 'Paper',
+    Scissors = 'Scissors'
+}
```

Comment:
Enums are great, they abstract away constantly having to reference a constant *and* you can change their value in one place and it'll be reflect elsewhere